README.md

```diff
@@ -54,6 +54,23 @@ tile = tiler.get_tile(0, (0, 0))
 ```python
 tiler.close()
 ```
+
+## Setup environment for development
+Requires poetry and pytest and pytest-watch installed in the virtual environment.
+
+```console
+$ git clone https://github.com/imi-bigpicture/opentile.git
+$ poetry install
+```
+
+Set the OPENTILE_TESTDIR environment variable to your test data directory. The tests for svs and ndpi uses [openslide testdata](https://openslide.cs.cmu.edu/download/openslide-testdata/). The test data for philips tiff is currently not publically avaiable.
```

Comment:
Now I have to read the code to know how to organize the slides. Would it be an improvement to:

- Imitate the folder structure from the link
- Add to the readme that the folder structure should be kept

Or shortterm/longterm improvement. Create dev-script that fetches the files you want and puts it in the folder structure you want and then the tests just runs?

Reply:
Added a script for downloading and/or checking openslide testdata.